CMakeLists.txt

```diff
@@ -1,5 +1,4 @@
-# cmake 3.18+ to have the ARCHIVE_EXTRACT sub-command for files
-cmake_minimum_required(VERSION 3.18)
+cmake_minimum_required(VERSION 3.21)
```

Comment:
We depend on JsRuntimeHost which already required 3.21. Updating this should have no impact on compatibility. 3.21 is required for `PROJECT_IS_TOP_LEVEL` cmake flag.